inst/include/common/information.hpp

```diff
@@ -149,6 +157,77 @@ class Information {
     this->random_effects_parameters.push_back(&re);
   }
 
+  /**
+   * Register a parameter name.
+   *
+   * @param p_name
+   */
+  void RegisterParameterName(std::string p_name) {
+    this->parameter_names.push_back(p_name);
+  }
```

Comment:
@Andrea-Havron-NOAA I thought that we had talked about removing this new capability of storing the parameter names in a vector? Although, it might be helpful for power users in the future and for debugging. What do you think @msupernaw and @rklasky?

Reply:
Oh right, I forgot to remove this code. I will wait for feedback before removing.

Reply:
I will file a new issue to address this topic.